spyder/workers/updates.py

```diff
@@ -76,7 +76,7 @@ def start(self):
                 data = page.read()
 
                 # Needed step for python3 compatibility
-                if not isinstance(data, str):
+                if not is_text_string(data, str):
```

Comment:
This is also wrong, and it's what's making our tests to time out in master because checking if there's a new Spyder version is failing now.